packages/dashboard/src/services/accountsApi.js

```diff
@@ -0,0 +1,5 @@
+import ky from "ky";
+
+const prefix = process.env.NEXT_PUBLIC_SKYNET_DASHBOARD_URL ?? "";
```

Comment:
Is this a new `.env` field that we need to add?

Reply:
not on production, only used locally to replace localhost with actual account.siasky.net or so